commit f647cd54c8fa2b0894f347cbf7f13aacc0b8c785
Author: Ake Sandgren <ake.sandgren@hpc2n.umu.se>
Date:   Tue Oct 18 14:30:08 2022 +0200

    Drop redundant setting of env var CXXFLAGS to what it already is.

diff --git xmipp-3.22.07-Helios/src/xmipp/SConstruct xmipp-3.22.07-Helios/src/xmipp/SConstruct
index c921c3c5..e70ea2cd 100644
--- xmipp-3.22.07-Helios/src/xmipp/SConstruct
+++ xmipp-3.22.07-Helios/src/xmipp/SConstruct
@@ -148,7 +148,6 @@ env['LINKERFORPROGRAMS'] = os.environ.get('LINKERFORPROGRAMS')
 env['CCFLAGS'] = os.environ.get('CCFLAGS', '').split()
 cxxFlags = os.environ.get('CXXFLAGS', '')
 env['CXXFLAGS'] = cxxFlags.split()
-os.environ['CXXFLAGS'] = cxxFlags # FIXME use only env or os.environ in the rest of the code
 env['LINKFLAGS'] = os.environ.get('LINKFLAGS', '').split()
 
 
